src/Types/ByteArray.js

```diff
@@ -0,0 +1,64 @@
+// From instance for Array in prelude
+exports.ordImpl = function (f) {
+    return function (xs) {
+        return function (ys) {
```

Comment:
A few stylistic nitpicks (if we feel these don't matter please feel free to disregard :)): 

- In the rest of the codebase, we use arrow functions for currying with FFI stuff 
- Sometimes `const` is used (and `let` where appropriate) in this module, but there's some `var`s here and below